src/app/js/formats/streamgraph/Streamgraph.js

```diff
@@ -382,16 +380,16 @@ class Streamgraph extends PureComponent {
                             return j != colorNameList.length - index ? 0.3 : 1;
                         });
 
+                    var currentLegendItem;
```

Comment:
it doesn't seem to be used in upper scopes, so I guess you can const it inside the loop